test/components/TxNotification.components.test.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import { render } from 'enzyme'
+
+import { TxNotification } from 'components/TxNotification'
+import { TX_HASH } from '../data'
+import { abbreviateString } from 'utils'
+import { Network, WalletInfo } from 'types'
+
+let networkId: Network = undefined
+
+jest.mock('hooks/useWalletConnection', () => {
```

Comment:
Is it posible to create a test also with `return { isConnected: false }`?

Reply:
Added.

Still, does it matter in that case?
From this component point of view, `networkId` is what we care about.
We know from the logic that if `networId` == `undefined` then `isConnected` is very likely `false`.

The reason for adding it in the mock is because this is a required property of `WalletInfo`.

Reply:
Yep, I meant to make some expectancies for when there's no connection (therefore networkId), but I see now u have them